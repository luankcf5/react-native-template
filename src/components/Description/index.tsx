import styled from "styled-components/native";

const Description = styled.Text`
  font-size: 14px;
  color: ${({ theme }) => theme.colors.description};
  margin-top: 2px;
`;

export default Description;
