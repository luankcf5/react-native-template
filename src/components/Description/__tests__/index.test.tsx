import * as React from "react";
import * as renderer from "react-test-renderer";
import WithTheme from "theme/withTheme";
import Description from "..";

describe("Description component UI", () => {
  test("Should render properly", () => {
    const tree = renderer
      .create(
        <WithTheme>
          <Description>Fastics Native Description</Description>
        </WithTheme>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
