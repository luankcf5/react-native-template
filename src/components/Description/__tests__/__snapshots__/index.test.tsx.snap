// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description component UI Should render properly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "gray",
        "fontSize": 14,
        "marginTop": 2,
      },
    ]
  }
>
  Fastics Native Description
</Text>
`;
